plugins/obs-filters/data/droste_filter.effect

```diff
@@ -0,0 +1,119 @@
+/*****************************************************************************
+Copyright (C) 2017 by c3r1c3 <c3r1c3@nevermindonline.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*****************************************************************************/
+
+/*
+ * "Droste Effect" for OBS by c3r1c3
+ * Inspired by https://www.shadertoy.com/view/4dGXD1
+ */
+
+/* Standard OBS->GPU uploads. */
+uniform float4x4 ViewProj;
+uniform texture2d image;
+
+/* Filter variables from OBS. */
+uniform float number_of_repeats;
+uniform float downscale_amount;
+uniform float x_offset;
+uniform float y_offset;
+uniform float downscale_inversion;
+
+/*
+ * Here we set how the base texture will be sampled/read/interpreted.
+ * Below is what you would normally use in OBS. Why?
+ * Linear filtering 'messes' with the image the least.
+ */
+sampler_state textureSampler {
+	Filter    = Linear;
+	AddressU  = Clamp;
+	AddressV  = Clamp;
+};
+
+/*
+ * Here we create the struct to hold the current position and texture info
+ * of the vertex in question.
+ */
+struct VertData {
+	float4 pos : POSITION;
+	float2 uv  : TEXCOORD0;
+};
+
+bool innerRectangle(float2 uv, float2 location, float2 size)
+{
+	bool test = false;
+	if (uv.x >= location.x && uv.x <= location.x + size.x &&
+		uv.y >= location.y && uv.y <= location.y + size.y) {
```

Comment:
Style:  Indent with two tabs for line continuations.

You established in an earlier review that long `if` statement conditions are indented with two tabs.